.github/CODEOWNERS

```diff
@@ -1 +1,18 @@
-* @iHiD
+# JS files
+*.js      @iHiD @kntsoriano @erikschierboom
+
+# Ruby files
+*.rb      @iHiD
+*.ru      @iHiD
+Gemfile*  @iHiD
+config/   @iHiD
+db/       @iHiD
+lib/      @iHiD
+
+# Docker and deployment files
+docker/   @iHiD
```

Comment:
Do you also want to add the `dev.Dockerfile` here?

Reply:
```suggestion
docker/          @iHiD
dev.Dockerfile   @erikschierboom @iHiD @kntsoriano
```